common/changes/@uifabric/fabric-website/jahnp-fix-tutorial_2018-11-03-20-51.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/fabric-website",
+      "comment": "Remove TypeScript from get started tutorial",
```

Comment:
nit: the changelog comment is not quite accurate

Reply:
Not anymore :) Thanks @cliffkoh , I'll fix this.

Reply:
I've also updated the title and description since this no longer removes the TypeScript recommendation.